sdk/keyvault/azsecrets/client.go

```diff
@@ -605,9 +676,15 @@ func (b *RecoverDeletedSecretPoller) pollUntilDone(ctx context.Context, t time.D
 	return recoverDeletedSecretResponseFromGenerated(b.recoverResponse), nil
 }
 
+// ResumeToken returns a token for resuming polling at a later time
+func (s *RecoverDeletedSecretPoller) ResumeToken() (string, error) {
+	return string(s.resumeToken), nil
```

Comment:
`resumeToken` is a string?
```suggestion
	return s.resumeToken, nil
```